src/__tests__/GameBoard.test.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import GameBoard from '../components/GameBoard';
+import { PlayerPiece } from '../PlayerPiece';
```

Comment:
Can this not be made available without an import?

Reply:
I don't know. Maggie helped me get the type definitions moved over to `@types/board.d.ts` but we weren't sure how to get the enum to come with it.